### PR TITLE
update template plugin

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
 include *.txt
 include *.rst
-include LICENSE
\ No newline at end of file
+include LICENSE
+include myplugin/protocols.conf
```
Comment (pconesa) [inline MANIFEST.in:4]: Should we add requirements.txt?

Comment (azazellochg) [reply MANIFEST.in:4]: it's kind of inexplicitly there: include *.txt

File: myplugin/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# **************************************************************************
+# Module to declare viewers
```
Comment (pconesa) [inline myplugin/tests/__init__.py]: Thi should say tests, right?

**Review by pconesa — CHANGES_REQUESTED** [review]
Minor changes. Good to create steps as methods not strings.

**Review by pconesa — APPROVED** [review]

**Pull request merged**